packages/office-ui-fabric-react/src/components/Dialog/docs/DialogBestPractices.md

```diff
@@ -0,0 +1,49 @@
+### Layout
+
+- Don't use more than three buttons.
+- Dialog boxes consist of a header, body, and footer.

```

Comment:
The article applies to all three listed parts, so it's not needed here. Otherwise, it would need to be applied to all three (e.g. "a header, a body, and a footer."), which isn't necessary for clarity.